.github/workflows/ci.yaml

```
@@ -0,0 +1,23 @@
+# Find full documentation here https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions
+name: CI
+
+on:
+  pull_request:
+
+  # Manual invocation.
+  workflow_dispatch:
+
+  push:
+    branches:
+      - main
+
+jobs:
+  CI:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "18.12.1"
```

You can get setup-node to read the .nvmrc (so that when upgrading you only have to edit in one place) https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#node-version-file
```suggestion
          node-version-file: '.nvmrc'
```